Dockerfile

```diff
@@ -24,6 +24,8 @@ ENV TERM=xterm
 RUN apt-get update && \
     apt-get install -y vim-tiny
 
+RUN pip install -e git://github.com/dinosk/reana-commons.git@user-mgmt#egg=reana-commons
```

Comment:
`s/dinosk/reanahub/g`

Reply:
`user-mgmt` ... I think the necessary changes were merged to `master` everywhere?

Reply:
👍 good catch